### ISSUE TITLE
[Summary Widget] Error persisting

### ISSUE THREAD

**Issue by vicfx** [nasa/openmct#1827]
1. Create a Summary Widget, "My Summary Widget"
2. Create two layouts, Layout A and Layout B. Add your summary widget to both.
3. Drag Layout B onto Layout A to initiate editing of Layout A.
   1. Navigate to Layout A by clicking on it in the tree
   2. Drag Layout B from the tree into the main viewing area of Layout A
   3. Layout B should appear in Layout A where dropped, and edit mode should be activated
4. Hit X to cancel editing.
5. __Observe:__ A persistence error is displayed in a dialog and the following appears on console:

```
(anonymous) @ :8000/node_modules/openmct/dist/openmct.js:32
http://localhost:8000/couch/openmct/331accf5-7f52-43bf-afd2-3e99dcf6ebac Failed to load resource: the server responded with a status of 409 (Conflict)
```

**Comment by VWoeltjen** [comment]
Already unable to reproduce using the same objects and the same steps. Timing may be at issue? The 409 is at least a clue, indicating that there's a [revision conflict](https://wiki.apache.org/couchdb/HTTP_Document_API#PUT) CouchDB is seeing

**Comment by ev1stensberg** [comment]
Coud I try to solve this? @VWoeltjen

**Comment by VWoeltjen** [comment]
Sure! That would be very much appreciated, since that has been difficult to reproduce.

**Comment by ev1stensberg** [comment]
Will do. Could you rephrase the 3rd step? As mentioned, unable to drag layer B into A without doing a right click and "programatically" moving it

**Comment by VWoeltjen** [comment]
@ev1stensberg Updated Step 3 above, hope that clears things up. I suspect this may only be reproducible when using CouchDB for persistence

**Comment by ev1stensberg** [comment]
@VWoeltjen Nice, thanks! Will look into it

**Comment by ev1stensberg** [comment]
Did you do anything on the CouchDB side when you got the error? I.e did you set up a db named `openmct` and made documents or anything in the db?

**Comment by VWoeltjen** [comment]
This was observed in a test environment with a previously-configured `openmct` DB, containing a variety of existing documents.

**Comment by ev1stensberg** [comment]
Could you send me some dummy documents per mail or similar for me to reproduce the test environment?

**Comment by larkin** [comment]
There was a partial fix in #1888, and this should be completely fixed after #1943.  Marking unverified, will close when #1943 is merged.

**Comment by larkin** [comment]
Testathon 2018-05-01: Verified.

**Comment by akhenry** [comment]
Testathon 2018-05-01: Did not observe any errant persistence messages. Verified fixed.
